fixpatches/src/commonMain/kotlin/com/saveourtool/sarifutils/adapter/SarifFixAdapter.kt

```diff
@@ -215,42 +227,89 @@ class SarifFixAdapter(
      * @param fileReplacementsList list of replacements from all rules
      * @param targetFiles list of target files
      */
-    private fun applyReplacementsToFiles(fileReplacementsList: List<FileReplacements>, targetFiles: List<Path>): List<Path> = fileReplacementsList.mapNotNull { fileReplacements ->
-        val targetFile = targetFiles.find {
-            val fullPathOfFileFromSarif = if (!fileReplacements.filePath.adaptedIsAbsolute()) {
-                fs.canonicalize(sarifFile.parent!! / fileReplacements.filePath)
+    @Suppress("MaxLineLength")
+    private fun applyReplacementsToFiles(
+        fileReplacementsList: List<FileReplacements>,
+        targetFiles: List<Path>,
+    ): List<Path> {
+        if (fileReplacementsList.isEmpty()) {
+            log.warn { "The list of replacements is empty." }
+        }
+        if (targetFiles.isEmpty()) {
+            log.warn { "The list of replacements is empty." }
```

Comment:
Probably wrong log message

Reply:
Yep.

Thanks for the heads-up.

Reply:
Addressed with 64af9402efb34d4e2754b300553ddbc85d38aeba